bin/derequire.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+//
+// wrapper for derequire because there's no simple CLI. grrrr.... >:(
```

Comment:
[this is false](https://github.com/calvinmetcalf/derequire/blob/master/bin/cmd.js) :smile: